### PR TITLE
[SDA-8287-FCT] Update api model to support audit log forwarding

### PR DESCRIPTION
**Pull request by davidleerh**

### CODE REVIEW

**Comment by davidleerh** [comment]
Update api model to propagate changes to ocm-sdk-go.
Pending update CHANGES

File: model/clusters_mgmt/v1/audit_log_type.model [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Contains the necessary attributes to support audit log forwarding
+struct AuditLog {
+	// ARN of the cw audit log forwarding role
```
Comment (andreadecorte) [inline model/clusters_mgmt/v1/audit_log_type.model:19]: cw -> CloudWatch

Comment (davidleerh) [reply model/clusters_mgmt/v1/audit_log_type.model]: Thank you.
Made the adjustment to replace the abbreviation.

**Pull request merged**